content/posts/2022/introducing-launch-on-reana.md

```diff
@@ -0,0 +1,94 @@
+hosted on external services directly from the REANA web interface, without
+relying on the `reana-client` CLI tool!
+
+<!--more-->
+
+## An overview on how to run workflows
+
+As you might already know, `reana-client` is the command-line tool that can be
+used to interact with the REANA platform. You can use it to create workflows, to
+upload files to their workspaces and to execute them. A tutorial on how to do so
+can be found in the page
+[First example](https://docs.reana.io/getting-started/first-example/) of the
+REANA documentation website.
+
+Another way to manage workflows is to use the Python API. This can be useful if
+you want to interact with REANA from your Python application or script. If you
+are interested in this topic, you should read the blog post on
+[How to run REANA workflows using Python API](/posts/2021/reana-client-python-api/).
+
+However, both of this approaches require installing `reana-client` and setting
+it up. They also require to have all the files needed by the workflow locally,
+so that they can be uploaded to its workspace. All of this can be a hassle in
+some cases, for example when users only want to run a workflow whose
+specification is published online.
+
+## Launching workflows from external sources
+
+If your workflow specification is hosted online, you can now launch it directly
+from the REANA's web interface. We currently support GitLab and GitHub
+repositories, but you can also use any URL that points to a zip archive or to a
+YAML specification file.
+
+For example, you can click
+[here](https://reana.cern.ch/launch?url=https%3A%2F%2Fgithub.com%2Freanahub%2Freana-demo-root6-roofit&name=reana-demo-root6-roofit)
+to launch the [RooFit demo](https://github.com/reanahub/reana-demo-root6-roofit)
+on the REANA instance at CERN. If you log in with a valid CERN account, you will
+be shown some information about the workflow to be run and a button to launch
+it:
+
+![Launch page](/images/launch-page.png)
+
+If you look at the address bar, this is the full URL of the page:
+
+```
+https://reana.cern.ch/launch?url=https%3A%2F%2Fgithub.com%2Freanahub%2Freana-demo-root6-roofit&name=reana-demo-root6-roofit
+```
+
+As you can see, everything is handled by the `/launch` endpoint, which executes
+the workflow specified by the `url` parameter. You can also provide a custom
+workflow name with `name`, choose a different specification file with
+`specification` and modify the parameters of the workflow with `parameters`.
+
+### Badges
+
+We have also prepared some badges that you can use to make your workflows
+runnable by everybody with a single click. Badges are just images that, when
+clicked, redirect the user to the launch page of REANA. This means that you can
+include them wherever you want, for example in the README of your Git repository
+or on your website. There are badges in the
+[Examples section](https://reana.io/#examples) of REANA's website and in the
+README files of all the demo repositories.
```

Comment:
*suggestion(non-blocking):* Maybe, link all the demo repositories to GitHub - https://github.com/search?q=org%3Areanahub+reana-demo-&type=Repositories. It will list all demo repositories for reanahub